README.md

```diff
@@ -642,6 +642,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**OpenKeychain**](https://github.com/open-keychain/open-keychain) <sup>**[[F-Droid](https://www.f-droid.org/app/org.sufficientlysecure.keychain)]**</sup>
 * [**Password Monitor**](https://github.com/StellarSand/Password-Monitor) <sup>**[[F-Droid](https://www.f-droid.org/app/com.password.monitor)]**</sup>
 * [**Vault**](https://github.com/arsvechkarev/Vault) <sup>**[[F-Droid](https://f-droid.org/app/com.arsvechkarev.vault)]**</sup>
+* [**ProtonPass**](https://github.com/protonpass/android-pass) <sup>**[[F-Droid](https://f-droid.org/packages/proton.android.pass.fdroid/)]**</sup>
```

Comment:
Thanks for your submission!

Please use the correct `/app/` url for F-Droid (see [contributing](https://github.com/offa/android-foss/blob/master/CONTRIBUTING.md)) and keep alphabetical order.